lib/common.sh

```diff
@@ -551,10 +556,10 @@ function pull_container_image_if_missing() {
 #
 # Kill and remove the infra containers
 #
-function remove_ironic_containers() {
+remove_ironic_containers() {
   #shellcheck disable=SC2015
   for name in ipa-downloader vbmc sushy-tools httpd-infra; do
-    sudo "${CONTAINER_RUNTIME}" ps | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" kill $name || true
-    sudo "${CONTAINER_RUNTIME}" ps --all | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" rm $name -f || true
+    sudo "${CONTAINER_RUNTIME}" ps | grep -w "${name}$" && sudo "${CONTAINER_RUNTIME}" kill "${name}" || true
```

Comment:
`&&` followed by `||` is risky, it'd be better written explicitly as:

```bash
if sudo "${CONTAINER_RUNTIME}" ps | grep -w "${name}$"; then 
  sudo "${CONTAINER_RUNTIME}" kill "${name}" || true
fi
```

Also `grep -q`, ie `grep -w -q` , no need to print out the match

Reply:
done